Forge/src/main/java/com/blamejared/crafttweaker/impl/util/WrappingBracketParser.java

```diff
@@ -0,0 +1,31 @@
+package com.blamejared.crafttweaker.impl.util;
+
+import com.blamejared.crafttweaker.api.zencode.bracket.IgnorePrefixCasingBracketParser;
+import org.openzen.zenscript.parser.BracketExpressionParser;
+
+import java.util.Collections;
+import java.util.Set;
+import java.util.function.BiConsumer;
+
+public final class WrappingBracketParser extends IgnorePrefixCasingBracketParser {
```

Comment:
Is this class something that can be moved to common, since the class exists on both Fabric and Forge side?